scrapers/ep_votes/models.py

```diff
@@ -150,10 +150,16 @@ class MemberInfo:
 
     @staticmethod
     def parse_full_name(name: str) -> Tuple[Optional[str], Optional[str]]:
-        first = r"(?P<first>[A-Za-z\-\'\s\.]+?)"
+        # Last names are indicated on the parliament with upper case letters.
+        # If we find two upper case letters following each other we found the end of the first name.
+        # There are also suffixes like "del" that are not part of the first name.
+        # Since names always start with a capital letter, we also found the end of the first name
```

Comment:
> Since names always start with a capital letter

Hier ist ein Vorname, der "de" enthält… 🙃 

<img width="260" alt="Screen Shot 2021-08-18 at 19 55 36" src="https://user-images.githubusercontent.com/1512805/129948251-97131731-f9a7-4ad5-a9d1-f84731a89694.png">